File: checkValidVoter.java
import java.util.Scanner;

public class checkValidVoter {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter your name: ");
        String name = sc.nextLine();
        System.out.print("Enter your age: ");
        int age = sc.nextInt();

        if (age >= 18) {
            System.out.print("Hello " + name + ", you are a valid voter.");
        } else {
            System.out.print("Sorry " + name + ", you can't cast the vote.");
            int yearsLeft = 18 - age;
            System.out.print("You will be eligible after " + yearsLeft + " years.");
        }
    }
}

File: python/checkValidVoter.py
# Check if user is a valid voter
name = input("Enter your name: ")
age = int(input("Enter your age: "))
if age >= 18:
    print(f"Hello {name}, you are a valid voter.")
else:
    print(f"Sorry {name}, you can't cast the vote.")
    years_left = 18 - age
    print(f"You will be eligible after {years_left} years.")
